index.html

```diff
@@ -30,11 +30,11 @@ <h1 class="add-shadow">Automate boring things!</h1>
 
 <div class="middle-layer">
   <div class="panel">
-    <h2>Open Source, extendable, scalable and simple</h2>
+    <h2>Open Source, Extendable, Scalable and Simple</h2>
     <p>Designed to take messages from chat services and execute Python functions (skills) based on their contents.</p>
   </div>
 
-  <h3 class="modules-intro">All your needs in one single framework</h3>
+  <h3 class="modules-intro">All your needs in one single framework...</h3>
```

Comment:
Not sure about this change, it seems it turns the sentence a bit vague which isn't desirable

Reply:
well as a newbie I would say it does have an effect on the viewers like me. and I basically did these changes to get the swag s...yeah... I won't lie

Reply:
I understand, although since we are a small project we can only give swag to PR's that fix an open issue on the main [opsdroid repo](https://github.com/opsdroid/opsdroid). Feel free to check the project and try to close one of the open issues to get some swag 👍